detection-rules/attachment_adobe_image_lure_fts.yml

```diff
@@ -0,0 +1,32 @@
+name: "Attachment: Adobe image lure with with suspicious link from first time sender"
+description: "Detects Adobe phishing messages with an Adobe logo attached, with suspicious link language
+  from a first-time sender."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and length(filter(attachments, .file_type not in~ ('png', 'jpeg', 'jpg', 'bmp'))) == 0
+  and length(body.links) >0
+  and all(body.links, .display_text is null)
+  and any(attachments,
+       beta.logo_detect(.).brand.confidence in ("high")
+      and beta.logo_detect(.).brand.name == "Adobe"
+      and any(file.explode(.),
+            any(.scan.strings.strings, strings.ilike(.,
+                "*review*", "*sign*", "*view*", "*completed document*", "*open agreement*"))
+      )
+  )
+  and (
+          (
+              sender.email.domain.root_domain in $free_email_providers
+              and sender.email.email not in $sender_emails
+          )
+          or (
+              sender.email.domain.root_domain not in $free_email_providers
+             and sender.email.domain.domain not in $sender_domains
+          )
+  )
```

Comment:
should we exclude emails from Adobe?

Reply:
Doesn't seem to be required. Adobe isn't really notorious for unsolicited spam, and legit emails won't match this criteria.